src/resolvers/user.ts

```diff
@@ -39,11 +40,13 @@ export function savedItems(
   args: {
     filter: SavedItemsFilter;
     sort: SavedItemsSort;
-    pagination: PaginationInput;
+    pagination: Pagination;
   },
   context: IContext
 ): Promise<SavedItemConnection> {
-  validatePagination(args.pagination);
+  validatePagination(args.pagination,
```

Comment:
todo: should assign it back to args.pagination.

Reply:
objects are passed by reference in javascript, this might be ok/

Reply:
ya thats what i thought too, but realized we have a logic in apollo-utils `when pagination== null, return {first:defaultSize}` and this won't get assigned back to our `args.pagination` 

https://github.com/Pocket/apollo-utils/blob/main/src/pagination/validatePagination.ts#L10

i can either do this or assign `pagination = {first: defaultSize`} in apollo-utils